WikiTemplate/Crypto/wikitemplate-cryptoiOS.md

```diff
@@ -0,0 +1,60 @@
+
+## Onboarding
+
+- [ ] Verify able to create a wallet by following onboarding
+- [ ] Verify not able to screenshots during onboarding screens
+- [ ] Verify able to backup wallet seed phrase during onboarding
+- [ ] Verify able to skip backup wallet seed screen to complete onboarding
+- [ ] Verify onboarding is shown when trying to interact with Dapp without creating wallet
+
+## Panel
+
+- [ ] Verify panel is only shown in URL when a Dapp is trying to interact
```

Comment:
Wallet notification is enabled by default, and wallet notification is only shown when a Dapp is trying to interact. Panel will be shown after user clicks the notification.

Reply:
Notification test is done in Dapp support [here](
https://github.com/brave/qa-resources/pull/470/files#diff-d3aef752b1d4ed033c22f1717619a31c37c43b4c517e460dcc5d53d53af7fa78R45) & [here](https://github.com/brave/qa-resources/pull/470/files#diff-d3aef752b1d4ed033c22f1717619a31c37c43b4c517e460dcc5d53d53af7fa78R46). 

This check is only for the wallet icon in URL